src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleDataprocDAO.scala

```diff
@@ -73,7 +74,7 @@ class HttpGoogleDataprocDAO(appName: String,
 
   }
 
-  override def deleteCluster(googleProject: GoogleProject, clusterName: ClusterName): Future[Unit] = {
+  override def deleteCluster(googleProject: GoogleProject, clusterName: ClusterName, region: Option[String]): Future[Unit] = {
     val request = dataproc.projects().regions().clusters().delete(googleProject.value, defaultRegion, clusterName.value)
```

Comment:
did you mean to pass `region` into here?

Reply:
Yes